Lib/gftools/fonts_public.proto

```diff
@@ -38,3 +38,7 @@ message AxisProto {
   optional float max_value = 4;
 };
 
+message SourceProto {
+  optional string repository = 1;
```

Comment:
repository_url if the expectation is it always will be one. If not I suggest a comment explaining what we will find here.

Reply:
Using repository_url